exercises/accumulate/AccumulateTests.elm

```diff
@@ -12,18 +15,21 @@ square x =
 
 tests : Test
 tests =
-    suite "Accumulate"
+    describe "Accumulate"
         [ test "[]] Accumulate"
-            (assertEqual [] (accumulate square []))
+            (\() -> Expect.equal [] (accumulate square []))
```

Comment:
I think this syntax is the convention the Elm Test community are trying to promote:

```elm
test "square Accumulate" <|
    \() -> 
        (accumulate square [ 1, 2, 3 ])
            |> Expect.equal [ 1, 4, 9 ] 
```